src/Components/Button.lua

```diff
@@ -127,6 +128,12 @@ function Button:render()
 					Elevation = 2;
 					_mouseOver = false;
 				})
+			end;
+			
+			[Roact.Event.MouseButton1Click] = function() --action's will allow users to make it so when the button is pressed it will do something. (idea: later on make it so the user can pick the actionType and is not limited to just MouseButton1Click?)
```

Comment:
The code is clear; it's obvious what's happening - the comment's unnecessary for describing what it's doing. I'm also not a huge fan of putting ideas in comments, because they tend to get missed: people don't read through the entire codebase just to see what needs to be done!